api/v1alpha1/databasesecretengineconfig_types.go

```diff
@@ -151,7 +159,6 @@ type DBSEConfig struct {
 
 	// VerifyConnection Specifies if the connection is verified during initial configuration. Defaults to true.
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=true
```

Comment:
I removed this line because with it, the created Config always had `verify_connection: True`. Without this line, the created resource reflects what is specified in the corresponding YAML.